### DISCUSSION TITLE
How to deploy a `@sveltejs/adapter-node` website to Digital ocean droplet?

### DISCUSSION THREAD

**Discussion by mylastore** [sveltejs/kit#3024] [Q&A]
How to deploy a `@sveltejs/adapter-node` website to a Digital Ocean droplet?

I have been uploading everything to the droplet and building everything on it. I am sure this is not the right way. 

Help, please.

**Comment by mgrubinger** [comment]
As as start, I would start by following a guide like this: https://www.digitalocean.com/community/tutorials/how-to-set-up-a-node-js-application-for-production-on-ubuntu-20-04
If you already built the project on the server, you can then tell pm2 to run `node build` (assuming you did not change the build directory) and have nginx serve up your application.
